README.md

```diff
@@ -163,3 +163,46 @@ In order to automatically format code in VS Code according to our style guide:
 2. Enable the **Editor: Format on Save** setting: `"editor.formatOnSave": true`.
 
 Your files will now be formatted automatically when you save them.
+
+### PR title rules
```

Comment:
Does this addition belong in the `CONTRIBUTING.md` instead?

We also have a (public!) Slab doc: https://beyond-essential.slab.com/posts/pr-titles-conventional-commits-avgsj3xb

Reply:
Ah good point, will move it there